WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -7,10 +7,11 @@ import Kingfisher
 ///
 final class BlazeCampaignDashboardViewHostingController: SelfSizingHostingController<BlazeCampaignDashboardView> {
     private let viewModel: BlazeCampaignDashboardViewModel
-    private let parentNavigationController: UINavigationController?
+    private let parentNavigationController: UINavigationController
     private lazy var blazeNavigationController = WooNavigationController()
+    private var coordinator: BlazeCampaignCreationCoordinator?
```

Comment:
Nit: You can declare the coordinator lazily to avoid optional here.

```suggestion
    private lazy var coordinator = BlazeCampaignCreationCoordinator(
            siteID: viewModel.siteID,
            siteURL: viewModel.siteURL,
            source: .myStoreSection,
            navigationController: parentNavigationController,
            onCampaignCreated: handlePostCreation
        )
```